bin/libs/common.ts

```diff
@@ -128,36 +128,21 @@ export function date(...args: string[]): string|undefined {
 
 
 let logExists = false;
-let logFile:std.File|null = null;
 
 function writeLog(message: string): boolean {
+  const filename = std.getenv('ZWE_PRIVATE_LOG_FILE');
+  if (!filename) {
+      return false;
+  }
+  logExists = fs.fileExists(filename);
   if (!logExists) {
-    const filename = std.getenv('ZWE_PRIVATE_LOG_FILE');
-    if (filename) {
+      fs.createFile(filename, 0o640, message);
       logExists = fs.fileExists(filename);
-      if (!logExists) {
-        fs.createFile(filename, 0o640, message);
-        logExists = fs.fileExists(filename);
-      }
-      if (logExists) {
-        let errObj = {errno:undefined};
-        logFile = std.open(filename, 'w', errObj);
-        if (errObj.errno) {
-          printError(`Error opening file ${filename}, errno=${errObj.errno}`);
-          logFile=null;
-          logExists=false;
-          return false;
-        }
-      }
-    }
-  }
-  if (logFile===undefined || logFile===null) {
-    return false;
   } else {
```

Comment:
If the log doesn't exist, we create it and drop the message with this new logic - the `else` is never hit on first pass. Is that intended?

Reply:
I believe, it is ok. We are creating file and initializing it with content of `message`. We have to skip the `else` part, otherwise we will duplicate the first message in the log.

Reply:
I see it now, my mistake. Thx!